hack/build.sh

```diff
@@ -9,8 +8,29 @@ BUILD_DATE=$(date --utc -Iseconds)
 
 mkdir -p bin
 
-LDFLAGS="-s -w "
-LDFLAGS+="-X github.com/medik8s/node-maintenance-operator/version.Version=${VERSION} "
-LDFLAGS+="-X github.com/medik8s/node-maintenance-operator/version.GitCommit=${COMMIT} "
-LDFLAGS+="-X github.com/medik8s/node-maintenance-operator/version.BuildDate=${BUILD_DATE} "
-GOFLAGS=-mod=vendor CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -ldflags="${LDFLAGS}" -o bin/manager main.go
+# first value must not be quoted
+LDFLAGS_VALUE="-X github.com/medik8s/node-maintenance-operator/version.Version=${VERSION}"
+# additional values must be quoted when used with GOFLAGS!
+LDFLAGS_VALUE+=" \"-w\""
+LDFLAGS_VALUE+="-X github.com/medik8s/node-maintenance-operator/version.GitCommit=${COMMIT} "
+LDFLAGS_VALUE+="-X github.com/medik8s/node-maintenance-operator/version.BuildDate=${BUILD_DATE}"
+# allow override for static flag
+LDFLAGS_STATIC="${LDFLAGS_STATIC:-" \"-s\""}"
```

Comment:
`-s` isn't for "static" but "symbol table", used for debugging. `-w` is for debugging too. I'd suggest to put both into `LDFLAGS_DEBUG`. WDYT?